mkdocs/docs/HPC/FAQ.md

```diff
@@ -217,12 +217,11 @@ See the explanation about how jobs get prioritized in [When will my job start](.
 
 {% else %}
 
-In practice it's
+In practice, it's
 impossible to predict when your job(s) will start, since most currently
-running jobs will finish before their requested walltime expires, and
-new jobs by may be submitted by other users that are assigned a higher
-priority than your job(s). You can use the `showstart` command. For more
-information, see .
+running jobs will finish before their requested walltime expires. 
+New jobs may be submitted by other users that are assigned a higher
+priority than your job(s). You can use the `showstart` command.
```

Comment:
Drop the mention of `showstart`, it no longer exists.

`squeue --start` can be used as a replacement though... Do also mention what it can be used for, for example:

```
You can use the `squeue --start` command to get an estimated start time for your jobs in the queue, but do keep in mind that it's just an unrealistic estimate.
```

Reply:
fixed in [78d4e07](https://github.com/hpcugent/vsc_user_docs/pull/657/commits/78d4e078ffa0aa4e73a24e12609addfae52b0c6a)